CHANGES.rst

```diff
@@ -6,10 +6,41 @@
 Summary
 +++++++
 
-- The Gammapy v0.11 release is planned for March 2019.
+- Released on March 29th, 2019
+- 11 contributors
+- 2 months of work
+- 61 pull requests (not all listed below)
 
 **What's new?**
 
+Gammapy v0.11 implements a large part of the new joint-likelihood fitting
+framework, proposed in `PIG 7`_. This includes the introduction of the
+`FluxPointsDataset`, `MapDataset` and `Datasets` classes, which now represent
+the main interface to the `Fit` class and fitting backends in Gammapy. As a
+first use-case of the new dataset classes we added a tutorial demonstrating a
+joint-likelihood fit of a CTA 1DC Galactic center observations. We also
+considerably improved the performance of the 3D likelihood evaluation (~factor
+of 10) by evaluating the source model components on smaller cutouts of the map.
+We also added a tutorial demonstrating the use of the `MapDataset` class for
+MCMC sampling and show how to interface Gammapy to the widely used emcee package.
+
+Gammapy v0.11 now supports the evalution of non-radially symmetric 3D background
+models and defining multiple background models for a single `MapDataset`.
+
+Gammapy v0.11 improves the support for the scipy and sherpa fitting backends. It
+now implements full support of parameter freezing and parameter limits for both
+backends. We also added a `reoptimize` option to the `Fit.likelihood_profile`
+method to compute likelihood profiles with reoptimizing remaining free parameters.
+
+For Gammapy v0.11 we added a SkyEllipse model to support fitting of elongated
+sources and changed the parametrization of the `SkyGaussian` to integrate correctly
+on the sphere. The spatial model classes now feature simple support for coordinate
+frames, such that the position of the source can be defined and fitted independently
+of the coordinate system of the data.
+
+Gammapy v0.11 also includes a new pulsar analysis tutorial. It demonstrates
```

Comment:
This sentence could be added at the end of the first paragraph, where you point out the other new tutorial ntoebooks.

Reply:
Done